mne/viz/_3d.py

```diff
@@ -3117,10 +3121,15 @@ def _plot_dipole(ax, data, vox, idx, dipole, gridx, gridy, ori, coord_frame,
 
     # These are the only two options
     coord_frame_name = 'Head' if coord_frame == 'head' else 'MRI'
-    plt.suptitle('Dipole #%s / %s @ %.3fs, GOF: %.1f%%, %.1fnAm\n%s: ' % (
-        idx + 1, len(dipole.times), dipole.times[idx], dipole.gof[idx],
-        dipole.amplitude[idx] * 1e9, coord_frame_name) +
-        '(%0.1f, %0.1f, %0.1f) mm' % tuple(xyz[idx]))
+
+    if title is None:
+        plt.suptitle('Dipole #%s / %s @ %.3fs, GOF: %.1f%%, %.1fnAm\n%s: ' % (
+            idx + 1, len(dipole.times), dipole.times[idx], dipole.gof[idx],
+            dipole.amplitude[idx] * 1e9, coord_frame_name) +
+            '(%0.1f, %0.1f, %0.1f) mm' % tuple(xyz[idx]))
+    else:
+        plt.suptitle(title)
```

Comment:
I would just do:
```
if title is None:
    title = ...
fig.suptitle(title)
```
It modernizes to use the class, plus is more DRY. But I can live with this variant

Reply:
Totally agree, just didn't dare to touch the existing `plt` use because I wasn't sure whether it's there for a reason. Will change tomorrow. :)

Reply:
Addressed in ba3e62e